vg/recorder/recorder.go

```diff
@@ -372,7 +373,7 @@ func (a *Fill) callerLocation() *callerLocation {
 type FillString struct {
 	Font   string
 	Size   vg.Length
-	X, Y   vg.Length
+	X, Y   vg.Length // FIXME: move to a vg.Point
```

Comment:
will do in the next rebase.
(it was quite a large CL already and I was getting a bit tired)